### PR TITLE
Fix security dependencies and add package.lock

### PR DESCRIPTION
**Pull request by michhyun1**
**Issue #:**

**Description of changes:**

Downgrade eslint to 7.x.x, since some the latest version of other deps have a peer dep of eslint 7.x.x

**Testing**

1. How did you test these changes?

npm i

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -205,19 +205,17 @@
     "electron-builder": "^22.14.5",
     "electron-devtools-installer": "^3.2.0",
     "electron-rebuild": "^3.2.2",
-    "eslint": "^8.4.1",
-    "eslint-config-airbnb-typescript": "^6.3.1",
-    "eslint-config-erb": "^0.3.0",
+    "eslint": "^7.32.0",
```
Comment (devalevenkatesh) [inline package.json:208]: Does this bring any other security alerts with the downgrade?

Comment (michhyun1) [reply package.json:208]: There are still some sec vulnerabilities but nothing critical

Comment (michhyun1) [reply package.json:208]: nevermind - I take that back - give me a few mins

Comment (devalevenkatesh) [reply package.json:208]: No worries, suggest just running `npm audit fix` or just do `npm update --save` to update all deps and see if we still have alerts and the demo runs fine.

Comment (michhyun1) [reply package.json:208]: Okay now there isnt!

**Review by devalevenkatesh — APPROVED** [review]

**Comment by devalevenkatesh** [comment]
Under "how did you test", could you please also add whether the demo runs fine after the `npm i` extra, some more information :)

**Pull request merged**